### PR TITLE
Multi windows

### PR DESCRIPTION
**Pull request by Ramjii**

### CODE REVIEW

File: test_plugins/UIPlugin.py [hunk 1]
```diff
@@ -191,6 +214,156 @@ def prefab_button_pressed_callback(button):
...
+            self.update_content(self.tab_button1)
+            self.update_content(self.tab_button2)
+
+        def tab2_callback(button):
+            self.tab_button2.selected  = True
+            self.tab_button1.selected  = False
+            self.tab2.enabled = True
+            self.tab1.enabled = False
+
+            n_node = self.tabs.create_child_node("AA")
```
Comment (ajm13) [inline test_plugins/UIPlugin.py]: this is doing weird stuff with the layout, not sure if intentional?

Comment (Ramjii) [reply test_plugins/UIPlugin.py]: Oops, that shouldn't be there, nice catch

File: nanome/_internal/_ui/_menu.py [hunk 2]
```diff
@@ -7,14 +7,15 @@ class _Menu(object):
     def _create(cls):
         return cls()
 
-    def __init__(self, title = "title"):
```
[the remark this answers is not in the document]
Comment (Ramjii) [reply nanome/_internal/_ui/_menu.py]: I wasn't sure. We're never using _menu directly, no?

[the remark this answers is not in the document]
Comment (Ramjii) [reply nanome/_internal/_ui/_menu.py]: Yep, or just leave it to 0 for main menu

[the remark this answers is not in the document]
Comment (EthanVanderHorn) [reply nanome/_internal/_ui/_menu.py]: Oof yeah the title and index can't be set because only the default constructor is called by _create. Ooops

**Pull request merged**